### ISSUE TITLE
Aborted (core dumped) problem with isothiazole rings

### ISSUE THREAD

**Issue by takacsgergely** [IanAWatson/Lilly-Medchem-Rules#5]
I experienced the following issue: "Aborted (core dumped)" at the next structures:
S12N(C)C(=NC=1N=C(C1=CC=CC=C1)N2C1C=CC=CC=1)C1C=CC=CC=1
S12SC(=CC=1C(=NN2C1C=CC(=CC=1)N(=O)=O)CCCCC(OC)=O)C(C)(C)C
S12=C(C(=NN1C1=CC=CC=C1)C)C(=CN2C1C=CC(=CC=1)OC)C
S12=C3C(=CN1NC1=CC=C(C=C1)N(=O)=O)CCCC3=NN2C1=CC=CC=C1
S12SC(=CC=1C(=NN2C1C=CC=CC=1)N=NC1C=CC=CC=1)C(C)(C)C
S12SC(=C(C)C=1C=C(C1C=CC=CN=1)S2)C
S12N(C3=CC=CC=C3)C(=NC=1N(C(C(Cl)(Cl)Cl)N2C(OC)=O)C)C1C=CC=CC=1
S1(CC2=CCS(=O)(=O)C2)NC2=C(C=CC=C2)N=1
For the other 300k molecules the program ran properly.

**Comment by IanAWatson** [comment]
Want to leave you the opportunity to comment.

**Comment by takacsgergely** [comment]
Thank you very much for your quick help. The issue was solved the programs
runs well. It isn't a very common structure indeed...

It was my pleasure, thank you for solving it.

And of course thank you for the porgram. It's very useful and easy-to use.

I noticed that since the update today it writes everything to stdout and
not into the output file. For me it's not a problem i can redirect it but
for others it might be a problem.

Thanks for your help

Greg

2018-01-15 3:46 GMT+01:00 Ian Watson <notifications@github.com>:
